defaults/main.yml

```diff
@@ -36,20 +36,22 @@ haproxy_type:
 # If we are of the revproxy type, what are the IPs that we should bind to
 haproxy_revproxy_bindips: ['*']
 
-# If we are of the ocp type, what are the IPs that we should bind to
-haproxy_ocp_bindips: []
 
-# The ansible group that contains nodes that we want to direct OCP master
-# traffic into the cluster We expect the `ip` attribute to be set in the
-# inventory hostvars, and we want this template to fail if we do not have this
-# set in inventory
-haproxy_ocp_master_group: ''
+# The environments dependent variables eg staging, prod etc
+ocp_envs:
+  - name: prod # The environment name
+    bindips: [] # If we are of the ocp type, what are the IPs that we should bind to for this environment    
+    # The ansible group that contains nodes that we want to direct OCP master
+    # traffic into the cluster We expect the `ip` attribute to be set in the
+    # inventory hostvars, and we want this template to fail if we do not have this
+    # set in inventory
+    haproxy_ocp_master_group: []
```

Comment:
and if we prefix in the list haproxy, we can then just use master_group ? (cosmetic)